Corzbank.Data/Entities/TransferCard.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Corzbank.Data.Entities
+{
+    public class TransferCard: KeyEntity
+    {
+        public int TransferId { get; set; }
+        public Transfer Transfer { get; set; }
+
+        public int SenderCardId { get; set; }
+        public int ReceiverCardId { get; set; }
+        public Card Card { get; set; }
```

Comment:
Wrong configuration. You should set each FK separately, [hint](https://stackoverflow.com/questions/5559043/entity-framework-code-first-two-foreign-keys-from-same-table)